monai/transforms/spatial/array.py

```diff
@@ -1530,7 +1533,8 @@ def __call__(
         Returns:
```

Comment:
Could you please help add the new arg to doc-string?

Thanks.

Reply:
Good point, done!